data/src/main/java/com/linkedin/data/schema/grammar/PdlSchemaParser.java

```diff
@@ -1093,6 +1094,10 @@ public DataSchema lookupName(String fullName)
     if (schema == null)
     {
       schema = getResolver().findDataSchema(fullName, errorMessageBuilder());
+      if (schema != null)
```

Comment:
This method is now completely identical to `AbstractSchemaParser#lookupName`. Why don't we just delete this one and use that?

Reply:
good idea